docs/content_management/data_migration/importing_data.md

```diff
@@ -213,6 +213,11 @@ The following example shows how to create a content type with two Field definiti
 
 The required metadata keys are: `identifier`, `mainTranslation`, `contentTypeGroups` and `translations`.
 
+The default values of Field definition properties mirror the underlying PHP API, for example:
+
+* `translatable` defaults to `true`
+* `required` defaults to `false`
```

Comment:
```suggestion
- `translatable` defaults to `true`
- `required` defaults to `false`
```

It is by no means a mistake, asterisk is a permissible sign here, but we tend to use minus sign for bullets throughout out docs.

Reply:
Thanks! Applied in https://github.com/ibexa/documentation-developer/pull/2349/commits/da9fb5d09d13cb854484e73f732631c3fe18be11 - and I will remember about it in the future (or, to put it another way, markdownlint will remember about it for me 😄 )